diagram-util/src/main/java/com/powsybl/diagram/util/forcelayout/ForceLayout.java

```diff
@@ -302,15 +314,10 @@ private void updateVelocity() {
     }
 
     private void updatePosition() {
-        // TODO do not compute forces or update velocities for fixed nodes
         // We have computed forces and velocities for all nodes, even for the fixed ones
```

Comment:
`// We have computed forces and velocities for all nodes, even for the fixed ones`
`// We can optimize calculations by ignoring fixed nodes in those calculations`

Are those comments up-to-date? Is velocity still computed for fixed points and is there still a possibility for optimization?